### PR TITLE
Wla 531

### PR DESCRIPTION
**Pull request by rishikrsharma**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
WLA-531

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/EvidenceShareFunctionalTest.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void processAnAppealWithValidMrn_shouldGenerateADl6AndAddToCcdAndUpdateSt
 
         SscsCaseDetails caseDetails = findCaseById(ccdCaseId);
         SscsCaseData caseData = caseDetails.getData();
-
+        caseData.setLanguagePreferenceWelsh("No");
```
Comment (SomeshAcharya) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/EvidenceShareFunctionalTest.java:35]: Do we need to explicitly set to "No"?

Comment (rishikrsharma) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/EvidenceShareFunctionalTest.java]: No, Its not required. Removed.

File: src/main/java/uk/gov/hmcts/reform/sscs/service/TemplateService.java [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
             } else {
-                return new Template(dl6TemplateName, "dl6");
+                return getTemplateByLanguagePreference(caseData, DocumentType.DL6);
             }
         }
         return null;
     }
+
+    private Template getTemplateByLanguagePreference(final SscsCaseData caseData, final DocumentType documentType) {
+        if (caseData.isLanguagePreferenceWelsh()) {
```
[the remark this answers is not in the document]
Comment (rishikrsharma) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/TemplateService.java]: Done

File: src/integrationTest/java/uk/gov/hmcts/reform/sscs/config/DocmosisTemplateConfigTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.rules.SpringClassRule;
+import org.springframework.test.context.junit4.rules.SpringMethodRule;
+
+import uk.gov.hmcts.reform.sscs.ccd.callback.DocumentType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.LanguagePreference;
+
+
+@RunWith(JUnitParamsRunner.class)
```
[the remark this answers is not in the document]
Comment (rishikrsharma) [reply src/integrationTest/java/uk/gov/hmcts/reform/sscs/config/DocmosisTemplateConfigTest.java]: done

**Pull request closed without merging**